docs/user/exercises/programming-integrated-code-lifecycle-repository-view.inc

```diff
@@ -0,0 +1,71 @@
+For the programming exercises, we use a version control system called **Git**. Git is a distributed version control system that allows you to track changes in your code.
+These changes are stored in a repository. A repository is a collection of files and directories that are tracked by Git.
+How to navigate through the repository and view the code is explained in the following sections. This guide is divided into two parts: one for students and one for instructors.
+It is shown on course programming exercises, but the same principles apply to exams as well.
+
+- **For Students:**
+
+As a student, you can use the repository to view your course submissions.
+To access your submissions, you need to navigate to the exercise an press the '*Open Repository*' button.
+
+.. figure:: programming/open-repository-button.png
+   :alt: Online Repository Button
+   :align: center
+
+   Online Repository Button
+
+After pressing the button, you will be redirected to the **Latest Commit** page of the repository.
+This view shows the latest commit in a read-only code editor. You can view the code, but you cannot edit it.
+You can also navigate through the whole code base by clicking on the file names in the left file directory.
+The right side of the screen problem statement. You can see the result of the submission. Lastly, you can also clone the repository to your local machine by pressing
```

Comment:
"The right side of the screen problem statement."

I don't understand, kollege